### DISCUSSION TITLE
how to configure once-daily payout?  is paymentProcessing interval actually milliseconds?

### DISCUSSION THREAD

**Discussion by mboyd1** [coinfoundry/miningcore#1354] [Q&A] [answered]
the default in my config under paymentProcessing was:
        // How often to process payouts, in milliseconds
        "interval": 600,

firstly, every 600 milliseconds seems awfully low.  do we really need to process payments that frequently?  is it perhaps in seconds and not milliseconds, i.e. 10 minutes (seems more reasonable).  the previous pool I operated did a once-daily payout, so I tried setting it to 24 hours in milliseconds:

"interval": 86400000,

but the software just crashes with this number

**Comment by seperatis** [comment, marked as answer]
"but the software just crashes with this number"
yeah, that's b/c you're setting payments to go every 24,000 hours.

600 **seconds**, eg, 10 minutes, **not milliseconds**. 

If you want to send payments **once per day**, then you can set `"interval": 86400`

**Comment by mboyd1** [comment]
ok so the units are in seconds, not milliseconds as the comments imply. thanks
